<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/BG.png);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <widget class="QLabel" name="logoLab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_StatCam1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Microsoft Yahei&quot;;font-size:10px;background:transparent;color:#ff0000;</string>
    </property>
    <property name="text">
     <string>摄像头1:故障</string>
    </property>
   </widget>
   <widget class="QLabel" name="_StatCam2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Microsoft Yahei&quot;;font-size:10px;background:transparent;color:#ff0000;</string>
    </property>
    <property name="text">
     <string>摄像头2:故障</string>
    </property>
   </widget>
   <widget class="QLabel" name="_StatNet">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: &quot;Microsoft Yahei&quot;;font-size:10px;background:transparent;color:#00ff00;</string>
    </property>
    <property name="text">
     <string>网络:正常</string>
    </property>
   </widget>
   <widget class="QWidget" name="rigitwidget" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>55</y>
      <width>282</width>
      <height>535</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/right.png);</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>272</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>270</width>
        <height>480</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="WorkStatusLab">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>25</y>
      <width>136</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/identify.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="leftwidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>55</y>
      <width>494</width>
      <height>535</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/left.png);</string>
    </property>
    <widget class="QWidget" name="widget2" native="true">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>275</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <widget class="QLabel" name="VideoImg2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget1" native="true">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <widget class="QLabel" name="VideoImg1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="ROILabel" name="VideoImgRoi1">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="ROILabel" name="VideoImgRoi2">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>275</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ROISetBtn">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>5</y>
      <width>135</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ResultTxt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="versionLab">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>5</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>V1.0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ROILabel</class>
   <extends>QLabel</extends>
   <header location="global">Controls/roilabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
